local/postman/Local Dev.postman_environment.json

```diff
@@ -0,0 +1,129 @@
+{
+	"id": "c99bfe48-fce6-47f2-8f4b-98a64f9dbecc",
+	"name": "Local Dev",
+	"values": [
+		{
+			"key": "CUSTOMER_PURIS_BACKEND_API_KEY",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "SUPPLIER_EDC_API_KEY",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "SUPPLIER_PURIS_BACKEND_API_KEY",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "SUPPLIER_PURIS_API_KEY",
```

Comment:
Great catch! This is mixed up. It should be SUPPLIER_EDC_API_KEY and the upper SUPPLIER_EDC_API_KEY should CUSTOMER_EDC_API_KEY.
You're still right that the CUSTOMER_EDC_API_KEY will currently not be used. Nevertheless I would keep it in case we need it in future / to reuse the environment maybe also for other tests :) Sidenote: These vars are currently used on authorization part of folders.